src/OrbitGl/App.cpp

```diff
@@ -138,7 +138,8 @@ bool DoZoom = false;
 OrbitApp::OrbitApp(MainThreadExecutor* main_thread_executor,
                    const orbit_base::CrashHandler* crash_handler,
                    orbit_metrics_uploader::MetricsUploader* metrics_uploader)
-    : main_thread_executor_(main_thread_executor),
+    : string_manager_(std::make_shared<StringManager>()),
```

Comment:
I see no reason, why StringManager is a shared_ptr in the first place. Could probably be just a value, not even a unique_ptr is needed.

(Sorry, unsolicited comment. Feel free to ignore.)

Reply:
No, good that you looked at it. I changed it to simple `StringManager string_manager_`. 
Since this is a fairly small PR, I amended, force pushed and will merge soon

Reply:
Thanks a lot for including this!